resources/buildConfigDefinitions.js

```diff
@@ -103,7 +106,7 @@ function processProperty(property, iface) {
   if (envLine) {
     env = envLine.split(' ')[1];
   } else {
-    env = (prefix + toENV(name));
+    env = prefix + toENV(name);
```

Comment:
Could you run `npm run prettier && npm run lint-fix`? It seems there were some style changes unrelated to this PR.

Reply:
Done !